Whiskey/Functions/New-WhiskeySemanticVersion.ps1

```diff
@@ -39,6 +44,21 @@ function New-WhiskeySemanticVersion
     {
         $semVersion = $Version | ConvertTo-WhiskeySemanticVersion -ErrorAction Stop
     }
+    elseif( $Path )
+    {
+        if($Path -like '*/package.json')

```

Comment:
1. Resolve the path so that if someone passes in a relative path, it gets converted to a full path, e.g `$Path = Resolve-Path -Path $Path | Select-Object -ExpandProperty ProviderPath`. Write an error if that returns nothing, e.g. `Path to version file '{0}' does not exist.' -f $PSBoundParameters['Path']).
2. Instead of `$path -like WILDCARD`, get a file info object so you can get directly at the filename and extensions, e.g.
```
    $fileInfo = Get-Item -Path $Path
    if( $fileInfo.FileName -eq 'package.json' )
    # ...snip...
   if( $fileInfo.Extension -eq '.psd1' )
```

Reply:
Done